serving/docker/tensorrt-llm.Dockerfile

```diff
@@ -64,7 +64,7 @@ RUN mv *.deb djl-serving_all.deb || true
 
 # Install OpenMPI and other deps
 ARG DEBIAN_FRONTEND=noninteractive
-RUN apt-get update && apt-get install -y g++ wget unzip openmpi-bin libopenmpi-dev libffi-dev git-lfs rapidjson-dev && \
+RUN apt-get update && apt-get install -y g++ wget unzip openmpi-bin libopenmpi-dev libffi-dev git-lfs rapidjson-dev graphviz && \
```

Comment:
Nit: why is the apt-version of the package necessary if we are also pip installing it?

Reply:
It is a system package with a python interface. The 2 need to be installed separately.

Reply:
ok... We can install this